.gitignore

```diff
@@ -7,7 +7,6 @@
 .*.sw*
 .*.un~
 clover.xml
-composer.lock
```

Comment:
@Maks3w Please see the changes in `.travis.yml`, which has been updated to use the lockfile so we can use the lowest / locked / latest strategy.